### ISSUE TITLE
[SUGGESTION] Visually differentiate (colorize) decorable/colorable brick sections (.g1, .g2 etc.) on canvas

### ISSUE THREAD

**Issue by fafayqa** [Polymaker/ldd-brick-editor#30]
If a brick has several g parts (that is more than 2: "main" g and 1 decor) it may become quite messy, like in this example where it is a bit harder to orient/differentiate between separate parts.

![01](https://github.com/Polymaker/ldd-brick-editor/assets/26612946/91420868-660e-4ec0-8969-db614f5203dc)

My suggestion is to introduce g parts colorization - just for the app canvas, of course, something like this:

![02](https://github.com/Polymaker/ldd-brick-editor/assets/26612946/ab49d2fd-7307-4aba-8623-9c09e983aa57)

Either having new settings section "Brick parts colorization", where one would manually define color for each brick part up to let's say 15th order (I guess that is the highest part section I was able to find, specifically brick 75972), or some auto-coloring code that would get all the brick section and define different color for each of them by the app itself thus no new settings section would be needed..

**Comment by Polymaker** [comment]
I completely agree and I had already thought of this feature but the rendering code is really messy and I could not find an easy way to make it work. It is far from impossible but I decided to put my time on other features instead.

I don't have an ETA but I will definitively add this feature at some point
